src/include/miopen/conv/invokers/impl_gemm_dynamic.hpp

```diff
@@ -29,12 +29,75 @@
 #include <miopen/invoker.hpp>
 #include <miopen/kernel.hpp>
 #include <miopen/conv/context.hpp>
+#include <miopen/numeric.hpp>
+#include <miopen/conv/asm_implicit_gemm.hpp>
 
 #include <vector>
 
 namespace miopen {
 namespace conv {
 
+struct _magic_div_u32_
```

Comment:
I've seen this in #680

Reply:
So this should be moved into a separate utility file and share it, right?

@atamazov @fronteer

Reply:
No. Just a warning. Now I see that #680 has is in the same file. Nothing to do.